tests/unit/packages/conftest.py

```diff
@@ -0,0 +1,44 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2022 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
```

Comment:
Craft-parts files should be LGPL instead of GPL.

Reply:
thanks that's a good catch. I think I copied this over from some other craft-parts file, so I'll look into fixing that in another PR (will fix these new headers here though)